plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java

```diff
@@ -54,7 +54,7 @@ public DeltaLakeTableProperties()
         tableProperties = ImmutableList.<PropertyMetadata<?>>builder()
                 .add(stringProperty(
                         LOCATION_PROPERTY,
-                        "File system location URI for external table",
+                        "File system location URI for the table",
```

Comment:
Can we update `delta-lake.rst` as well? 
>   * - ``location``
    - File system location URI for the external table.

Reply:
Good catch, updated